src/ImageSharp/Processing/Processors/CloningImageProcessor{TPixel}.cs

```diff
@@ -160,5 +167,30 @@ protected virtual void AfterImageApply(Image<TPixel> destination)
         protected virtual void Dispose(bool disposing)
         {
         }
+
+        private Image<TPixel> CreateTarget()
```

Comment:
This method was identical in every implementation. The only different was where the target size came from.